packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -6,9 +6,10 @@
  *
  * @emails react-core
  */
-
 'use strict';
 
+import hasOwnProperty from 'shared/hasOwnProperty'
```

Comment:
Don't do this in tests. This prevents us from being able to test compiled bundles, or at least introduces difficulties.

Reply:
Ok i will revert this